src/components/Sidebars/InnerSidebars/InnerGardenNavigationSidebar.js

```diff
@@ -18,25 +14,19 @@ const InnerGardenNavigationSidebar = ({
   onToggle,
   onOpenCreateProposal,
 }) => {
-  const location = useLocation()
-  const { preferredNetwork } = useWallet()
-
-  const connectedGarden = useConnectedGarden()
-  const networkType = getNetworkType(
-    connectedGarden?.chainId || preferredNetwork
-  )
```

Comment:
Reverted this back as `buildGardenPath` already handles the network type in the regex, and we can assume that the current location will have a garden path since this is rendered